src/server.py

```diff
@@ -37,6 +52,21 @@ def api_top_image():
         "topimage": article.top_image}), 200, {'Content-Type': 'application/json'}
 
 
+def fetch_og_tags(url):
+    title, description, imageUrl = web_preview(url, timeout=20)
```

Comment:
Are we using different library now? Doesn't newspaper library provide the same functionality?

Reply:
Apparently, newspaper doesn't have any option that we can use to get desired og tags 
And yes, web_preview is another one which can help us to do that, please have a look here https://github.com/ludbek/webpreview

Reply:
Thanks for the explanation!